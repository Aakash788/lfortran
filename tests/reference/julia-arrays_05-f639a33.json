{
    "basename": "julia-arrays_05-f639a33",
    "cmd": "lfortran --no-color --show-julia {infile}",
    "infile": "tests/../integration_tests/arrays_05.f90",
    "infile_hash": "09eac1191b2e8a76fafedf1307b2d1240ffbc5f632c08afa847df805",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "julia-arrays_05-f639a33.stdout",
    "stdout_hash": "df4b90eba03aa215e4e0d200a621a02bc60ac9dbd9d8bf8eaad40dc1",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}