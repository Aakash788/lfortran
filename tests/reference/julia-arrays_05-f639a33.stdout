function main()
    local i::Int32
    local numbers::Array{Float32, 1} = [1.500000e+00, 3.200000e+00, 4.500000e+00, 9.000000e-01, 7.200000e+00]
    for i âˆˆ 1:5
        numbers[i] = Float32(i) *   2.00000000000000000e+00
    end
    numbers .= [1.500000e+00, 3.200000e+00, 4.500000e+00, 9.000000e-01, 7.200000e+00]
end

main()
